exe/secret_env

```diff
@@ -2,7 +2,7 @@
 
 require "secret_env"
 require 'open3'
-config, env = ARGV.shift(2)
+config, env = ENV.key?('SECRET_ENV') ? ARGV.shift(1).push(ENV['SECRET_ENV']) : ARGV.shift(2)
```

Comment:
Command arguments should have higher priority than environment variables.
Take an environment variable only if a command argument env is not found in config.

Reply:
Fixed with your advice
@adorechic PTAL